stage-refractr.yml

```diff
@@ -19,6 +19,9 @@ refracts:
 
 - mozilla.org/: refractr2.allizom.org
 
+- m.youtube.com/: ftv.stage.mozaws.net # SVCSE-350 FTV redirect to youtube
```

Comment:
are you intending to redirect the mozaws just to youtube in general, or some specific video/channel at youtube?

and, do you want to direct to the mobile version of youtube?

Reply:
This is a migration from an existing redirect using redirect rules on an S3 bucket /w the following rules:

The intent is to simplify this and redirect everything going to `ftv.stage.mozaws.net` instead of having rules for `http://ftv.stage.mozaws.net/ytht*` & `http://ftv.stage.mozaws.net/ytfc*`. There is no content being served out of this bucket and anything that does not match these two rules results in a 403. 

```
[
    {
        "Condition": {
            "KeyPrefixEquals": "ytht"
        },
        "Redirect": {
            "HostName": "m.youtube.com",
            "HttpRedirectCode": "302",
            "Protocol": "https",
            "ReplaceKeyWith": ""
        }
    },
    {
        "Condition": {
            "KeyPrefixEquals": "ytfc"
        },
        "Redirect": {
            "HostName": "m.youtube.com",
            "HttpRedirectCode": "302",
            "Protocol": "https",
            "ReplaceKeyWith": ""
        }
    }
]
```

Reply:
I guess I follow that getting rid of an s3 bucket and a cloudfront is good, I'm not really following what purpose the redirect has. Pretty atypical for us to redirect to the homepage of some externally owned website. Why not just delete the bucket/cloudfront/etc. etc. ?

Reply:
This is the bug for having it set up originally. [FTV Redirector URL](https://bugzilla.mozilla.org/show_bug.cgi?id=1434653)

There are still ftv clients hitting `ftv.prod.mozaws.net/ytfc` and `ftv.prod.mozaws.net/ytht` and getting the appropriate redirect to `m.youtube.com.` Anything else will result in 403s. 

Since we are not serving any other content, I had planned on simplifying the config and just redirecting `ftv.prod.mozaws.net` & `ftv.stage.mozaws.net` rather than creating rules to redirect the two specific paths required. 


I have already modified the DNS record for `ftv.stage.mozaws.net` to point to refactr stage in preparation for this change so it is currently not functioning, but you can test with `ftv.prod.mozaws.net/ytfc`

```
curl -v ftv.prod.mozaws.net/ytfc
*   Trying 13.224.14.100...
* TCP_NODELAY set
* Connected to ftv.prod.mozaws.net (13.224.14.100) port 80 (#0)
> GET /ytfc HTTP/1.1
> Host: ftv.prod.mozaws.net
> User-Agent: curl/7.64.1
> Accept: */*
>
< HTTP/1.1 302 Moved Temporarily
< Content-Length: 0
< Connection: keep-alive
< Date: Wed, 02 Feb 2022 21:05:03 GMT
< Location: https://m.youtube.com/
< Server: AmazonS3
< X-Cache: Hit from cloudfront
< Via: 1.1 ee8246c5442dace7525c74f6a799bb46.cloudfront.net (CloudFront)
< X-Amz-Cf-Pop: SEA19-C2
< X-Amz-Cf-Id: ecMlSAox6D4-zrP_SA4aBUqcfyGld7GzbsoiTHOI0bruQxxs7H_dEg==
< Age: 77939
<
* Connection #0 to host ftv.prod.mozaws.net left intact
* Closing connection 0
```

Reply:
Sounds good - thank you for explaining.